pipeline/src/node_properties/track_properties.rs

```diff
@@ -20,3 +22,17 @@ pub enum Tracker {
     /// bounding boxes across frames to determine the object’s unique ID.
     Iou,
 }
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(tag = "type", rename_all = "snake_case")]
```

Comment:
As briefly discussed on Slack, this enum shouldn't have any serde attributes.

Reply:
Actually, `rename_all = "snake_case"` should probably stay (since we usually use lowercase string enum values).